integration/src/main/resources/configs/integration/ReferencedTerraWorkspaceLifecycle.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
This test will not get run automatically until you add an entry into the `FullIntegration` test suite for the environments where you want it tested. I wonder if we have other "orphan" tests.

Reply:
Oops, done. :)